packages/client/src/game/Game.ts

```diff
@@ -34,16 +34,34 @@ export class Game extends EventBus {
   private table!: TableEntity;
 
   private clockwiseMovement = true;
+  private gameStatus = false;
 
   constructor() {
     super();
   }
 
-  startGame(playersNum: number, playerData: GamePlayerType) {
-    this.players = addPlayers(playersNum, playerData);
+  startGame(playersNum: number, playerData?: GamePlayerType) {
+    this.gameStatus = true;
+    /* Если есть данные, добавляем игрока с этими данными */
+    if (playerData !== undefined) {
+      this.players = addPlayers(playersNum, playerData);
+    }
+
+    /* Если ранее игроки были добавлены и количество игроков изменилось */
+    /* Добавляем игроков заново с данными юзера, используемыми ранее */
+    if (this.players.length && this.players.length !== playersNum) {

```

Comment:
Мне кажется достаточно
```suggestion
    if (this.players.length !== playersNum) {

```

Reply:
да, согласна, без этой проверки всё сработает так же корректно. Убрала